docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.ts

```diff
@@ -0,0 +1,63 @@
+import { ChangeDetectionStrategy, Component, ElementRef, Input, Renderer2, ViewChild } from '@angular/core';
+import { TooltipDirective } from '@ux-aspects/ux-aspects';
+
+@Component({
+    selector: 'uxd-icon-preview',
+    templateUrl: './icon-preview.component.html',
+    styleUrls: ['./icon-preview.component.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class IconPreviewComponent {
+    @Input() name: string;
+    @Input() classname: string;
+    @Input() iconset: string;
+    @Input() iconClass: string;
+
+    public tooltipText: string = 'Click to copy icon text';
+    public copied: boolean = false;
+
+    hideTriggers: string [] = ['mouseleave', 'blur'];
+    showTriggers: string [] = ['focus', 'mouseenter'];
+
+    @ViewChild(TooltipDirective, { static: true }) tooltip: TooltipDirective;
+
+    constructor(
+        private readonly _renderer: Renderer2,
+        private readonly _elementRef: ElementRef
+    ) {}
+
+    copy(): void {
+
+        const dummy = this._renderer.createElement('input');
+        this._renderer.appendChild(this._elementRef.nativeElement, dummy);
+
+        switch (this.iconset) {
+            case 'ux-icons':
+                dummy.value =  '<i class="ux-icon' + ' ' + this.classname + '"></i>';
```

Comment:
There seems to be a double space after the equal sign, there also seems to be an unnecessary space: `+ ' ' +`. We can use backticks to make this easier: 

```typescript
dummy.value = `<i class="ux-icon ${this.classname}"></i>`;
```

Reply:
changed to use <code>${this.classname}</code>